packages/nodes-base/credentials/WiseApi.credentials.ts

```diff
@@ -30,5 +30,15 @@ export class WiseApi implements ICredentialType {
 				},
 			],
 		},
+		{
+			displayName: 'Private Key (Optional)',
+			name: 'privateKey',
+			type: 'string',
+			default: '',
+			description: 'Optional private key used for Strong Customer Authentication (SCA)',
```

Comment:
Would be useful to mention here the two SCA operations.
```suggestion
			description: 'Optional private key used for Strong Customer Authentication (SCA). Only needed for X and Y.',
```

Reply:
Updated to `Only required to retrieve statements, and execute transfers`